app/models/assignment.rb

```diff
@@ -6,4 +6,7 @@ class Assignment < ActiveRecord::Base
   belongs_to :employee
 
   scope :upcoming, -> { where('week_id >= ?', Week.upcoming.first.id) }
+
 end
+
+ASSIGNMENTS_PER_WEEK = 2
```

Comment:
I was trying to communicate that this should go inside the Assignment class (right now it is a global constant that just happens to be defined in this file) but now I think it should be in the Week class: `Week::ASSIGNMENTS_PER_WEEK`.